doc/main.dox

```diff
@@ -116,7 +119,7 @@ sockperf: [tid 4805] using epoll() to block on socket(s)
 @code
  -h,-?   --help,--usage         -Show the help message and exit.
          --tcp                  -Use TCP protocol (default UDP).
```

Comment:
I think that it will be useful to add a synonym `--stream` for `--tcp` to be consistent with UDS terminology.

Reply:
Done

Reply:
It will be good to change description too. For example, "Use stream socket/TCP protocol (default is diagram socket/UDP protocol)".